src/components/elements/card.js

```diff
@@ -1,10 +1,59 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
-const Card = ({ text }) => <article>{text}</article>
+const CardWrapper = styled.article`
+    width: 328px;
+    min-height: 356px;
+    box-sizing: border-box;
+    border-radius: 4px;
+    box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.1);
+    background-color: var(--color-white);
+    padding: 4rem 5.6rem 4.6rem 5.6rem;
+    position: relative;
+    margin: 0 1rem;
+    ${props => props.firstCard && 'margin: 0 1rem 0 0'};
```

Comment:
I think it's cleaner to make a parent component wrapper to handle this and use `first-child:` and `last-child:`

could be: 

```
<Card.Group>
   <Card />
   <Card />
   <Card />
</Card.Group>
```